static/docker-compose.yml

```diff
@@ -250,6 +251,7 @@ volumes:
   dbdata: {}
   uploads: {}
   letsencrypt: {}
+  localesOverride: {}
```

Comment:
This syntax will result in a volume (that is _not_ a bind mount) using the `local` driver.
This will allow files to persist between container restarts without depending on the folder structure of the host machine's filesystem.